### DISCUSSION TITLE
Is it possible to have TS0044 (Tuya 4-button remote) directly control another device?

### DISCUSSION THREAD

**Discussion by floatiepen** [Koenkk/zigbee2mqtt#14360] [Devices]
If it is possible, is there some refernce on how to set it up (in the bind or scene tab?)

I can see the events in MQTT (and therefore also in Home Assistant), and can automate from there, but I'm wondering if it is possible to bind a button press to toggle another zigbee device directly, without making the round-trip through MQTT and HA?

It has happened a few times that HA isn't running, but the power is on.  I'd like the button to still work.

Thanks!

-John

**Comment by vil1driver** [comment]
Hi, I hope too...
